### DISCUSSION TITLE
Rain!

### DISCUSSION THREAD

**Discussion by cwagz** [jasonacox/Powerwall-Dashboard#60] [General]
Seems like the cleaning my panels got last night from the rain might be worth about 700 to 900 watts on todays peak.

Anyone else have a similar experience? This was probably the first good cleaning they have had this season.

**Comment by jasonacox** [comment]
Hi Chad, yes, we see 10-20% improvement in kWh/day after cleaning.  I do see a bump on peak too.

When the Tesla installers were here, I asked them about cleaning.  They suggested every 6-12 months and to use "only water".  A few months after the install, I noticed that the fires in our area had left a layer of ash on the panels.  I decided to clean them myself.  I set up the ladder on our 2 story house and hauled the water hose up to the root.  My wife helped.  She sat in the back yard in a lawn chair with her phone, "ready to call the ambulance."  I admit, there were a few heart stopping moments as I juggled the hose, mop and squeegee. I lost my footing on the wet roof.  Thankfully, I caught myself before I went over the edge.  My wife was already dialing 911. I got back up and managed to finish the "job" but vowed to explore solar panel cleaning companies if I survived this adventure.

We had a "mud rain" not too long ago that just left everything covered in dust.  It looked pretty terrible.  I had found a company that would clean our panels for $120.  Reflecting back on my DIY cleaning adventure, I gladly opened up my wallet.  They did a phenomenal job and even sent before/after pictures.  This will show the level of the "dirty" which wasn't terrible, but the end result was beautiful:

<img width="364" alt="image" src="https://user-images.githubusercontent.com/836718/183259600-dfbe907d-9223-4622-8255-ae50a7477e5a.png">
<img width="362" alt="image" src="https://user-images.githubusercontent.com/836718/183259622-e9a00756-08e0-4117-89ed-360d69503aaa.png">
<img width="364" alt="image" src="https://user-images.githubusercontent.com/836718/183259640-b17884f2-f2cc-45f8-95f7-5969381a17b7.png">
<img width="367" alt="image" src="https://user-images.githubusercontent.com/836718/183259657-2bcc21d6-aaf7-4db1-b168-3f2ebd7497ca.png">

The cleaning did have a positive impact in our daily kWh daily production, from 43kWh to 48kWh (~12% increase).  

<img width="531" alt="image" src="https://user-images.githubusercontent.com/836718/183260841-76d7e9ea-863e-46ba-a874-b65e623bcdc6.png">


I didn't expect to see a lot.  I haven't done a lot of research, but a quick scan of studies shows that dirty panels can drop production 5-25%.  I did find this from [UC San Diego](https://ucsdnews.ucsd.edu/pressrelease/cleaning_solar_panels_often_not_worth_the_cost_engineers_at_uc_san_diego_fi#:~:text=Researchers%20found%20panels%20that%20hadn,7.4%20percent%20of%20their%20efficiency.):

> "Researchers found panels that hadn't been cleaned, or rained on, for 145 days during a summer drought in California, lost only 7.4 percent of their efficiency."

I  think the positive part of that is that even dirty panels are doing an amazing job at generation.  But, I must confess, my motivation for cleaning is somewhat more vanity driven.  I love the hi-tech look of the clean panels.  I think we probably wash our cars for the same reason (less about efficiency) but I will also keep hoping and praying for rain to do the work for me. Otherwise, I have a good company to do the cleaning, because I won't be juggling that stuff up on the roof again. 😄

**Reply by cwagz** [reply]
This is great information. Glad you didn't go over the edge! I guess I shouldn't be surprised by the gain from a cleaning. I was just hoping to never have to actually think about cleaning the panels. After seeing the improvement, it will be hard to ignore though. My highest producing day was 6/18/22 with 78kWh. Right after the early AM rain on 8/5/22 I am making 70kWh again. It looks like I probably could have produced a lot more all spring / summer if I would have cleaned the panels after the Santa Ana winds blew dirt all over everything earlier this year.

No way I would attempt to clean mine up on my roof. I probably would have gone over the edge.

I am going to start looking for a company to do the cleaning just in case our next rain is too far away.

![image](https://user-images.githubusercontent.com/13752647/183271143-89d7134b-a86e-41ec-b68c-875cec1dcca0.png)

**Reply by jasonacox** [reply]
Nice!  Great graph.  As to the wish to have done this earlier, I had the same thought.  I know they were dirty during the highest peak solar days.  We maxed out at 51 kWh/day but wonder where it could have been if they were clean.  

I saw on google that the typical cost is $3-$10/panel. I can confirm that. I'm in Santa Clarita Valley and found Harold @ [Clarity Solar Solutions](https://www.claritysolarsolutions.com/) and their rate is $6/panel. I asked him to also clean the Tesla skirt/frame.  No extra charge but it sure makes it look nice. Worth asking if you have the skirts.
